assisted_deployment.sh

```diff
@@ -208,11 +213,30 @@ spec:
    requests:
     storage: 8Gi
 EOF
+}
+
+
+function deploy_assisted_operator() {
+  echo "Installing assisted-installer operator..."
+
+  generate_assisted_operator
+
+  oc apply -f "${ASSETS_DIR}/04-assisted-service.yaml"
+  wait_for_crd "agentserviceconfigs.agent-install.openshift.io"
+
+  oc apply -f "${ASSETS_DIR}/05-assisted-service-config.yaml"
+}
+
+
+function patch_extra_host_manifests() {
+  cp "${OCP_DIR}/extra_host_manifests.yaml" "${ASSETS_DIR}/06-extra-host-manifests.yaml"
 
+  sed -i s/"namespace: openshift-machine-api"/"namespace: ${ASSISTED_NAMESPACE}"/ "${ASSETS_DIR}/06-extra-host-manifests.yaml"
```

Comment:
Could we also add the inspect annotation and disable the cleaning mode?

Reply:
Yes, but I'd rather do it in a separate PR. It won't be the prettiest code ever (looking at you, Bash), so I'd rather keep it out of this one

Reply:
I have some commits in https://github.com/openshift-metal3/dev-scripts/pull/1193/commits which may help, I can rebase that or break out some of the commits if you'd like to use that approach vs modifying in assisted_deployment.sh